docs/installation.md

```
@@ -58,11 +61,14 @@ function install_mcpd() {
 ```
 
-!!! info "macOS Gatekeeper quarantine"
-    If you're on macOS, remove the quarantine flag before running `mcpd`:
-    ```
-    xattr -d com.apple.quarantine mcpd
-    ```
+{% hint style="info" %}
+**macOS Gatekeeper quarantine**
+
+If you're on macOS, remove the quarantine flag before running `mcpd`:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)